cpp/ql/src/semmle/code/cpp/Preprocessor.qll

```diff
@@ -214,7 +214,11 @@ class PreprocessorUndef extends PreprocessorDirective, @ppd_undef {
  * A C/C++ preprocessor `#pragma` directive.
  */
 class PreprocessorPragma extends PreprocessorDirective, @ppd_pragma {
-  override string toString() { result = "#pragma " + this.getHead() }
+  override string toString() {
```

Comment:
This new case handles the `_Pragma` directive for inline pragmas. What should the extractor emit for the "head" here?